website/docs/docs/collaborate/build-and-view-your-docs.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Build and view your docs with dbt Cloud"
+id: "build-and-view-your-docs"
+description: "Automatically generate project documentation as you run jobs."
+pagination_next: null
+---
+
+<p style={{ color: '#808080', fontSize: '1.1em' }}>
+dbt Cloud enables you to generate documentation for your project and data platform. The documentation is automatically updated with new information after a fully successful job run, ensuring accuracy and relevance.
+</p>
+
+The default documentation experience in dbt Cloud is [dbt Explorer](/docs/collaborate/explore-projects), available on [Team or Enterprise plans](https://www.getdbt.com/pricing/). Use [dbt Explorer](/docs/collaborate/explore-projects) to view your project's resources (such as models, tests, and metrics) and their lineage to gain a better understanding of its latest production state.
```

Comment:
@cafzal reworked this page to add explorer content here as the default experience. then included a dbt docs section lower in the page - what do you think?